x/auth/ante.go

```diff
@@ -66,64 +67,48 @@ func NewAnteHandler(am AccountMapper, fck FeeCollectionKeeper) sdk.AnteHandler {
 		if err != nil {
 			return newCtx, err.Result(), true
 		}
-
-		sigs := stdTx.GetSignatures() // When simulating, this would just be a 0-length slice.
```

Comment:
simply renamed to stdSigs to distinguish from it being the actual cryptographic signature as was previously implied.